src/styles/_catalog.scss

```diff
@@ -19,4 +20,24 @@
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  @media(max-width: 1023px) {
+    grid-template-columns: repeat(3, 1fr);
+
+    &__card:not(:nth-child(3n)) {
+      margin-right: 48px;
```

Comment:
You don't need margin right if you use grid. Just set gap: 48px in catalog class